notary_now_api/models.py

```diff
@@ -79,7 +79,7 @@ class Notary(models.Model):
     user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, related_name='notary_user')
     radius = models.PositiveIntegerField()
     active = models.BooleanField(default=True)
-    bio = models.TextField(max_length=5000)
+    bio = models.TextField(default="", max_length=5000)
```

Comment:
✅ Default bio to empty string; I agree, this makes sense to have the bio be an empty string